index.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="pages/index.css">
-  <title>Pink App</title>
+  <title>приложения Pink</title>
```

Comment:
@NazarPauk, I would suggest reviewing all the page head titles and deciding on a consistent naming convention because, currently, different patterns are being used.

```HTML
  # index.html
  <title>приложения Pink</title>

  # form.html
  <title>Пинк: Конкурс</title>
  
  # catalog.html
   <title>Pink: Фотографии</title>

  # components.html
  <title>UI Components - приложение Pink</title></head>
```

Reply:
Done✅